docs/installation.md

```diff
@@ -17,6 +17,14 @@ On other platforms, install `jrnl` using [Python](https://www.python.org/) 3.6+
 pipx install jrnl
 ```
 
+!!! note
+     `pipx` should be installed through `pip`. Missing dependencies and other issues
+     may occur when installing `pipx` through `apt` or another package manager.
+     
+!!! note
+     If you experience path issues after installing `jrnl`, ensure that the previous
```

Comment:
Can we say this in a more direct way? Maybe something like "Don't use sudo to install jrnl" or something along those lines.